src/targets/browser/index.jsx

```diff
@@ -25,14 +24,13 @@ const renderApp = () => {
     root.render(<Spinner size="xxlarge" middle={true} />)
     return
   }
-
-  const App = require('containers/App').default
+  const AnimatedWrapper = require('components/AnimatedWrapper').default
```

Comment:
Why do we want a `require` here instead of an import?

Reply:
But after a quick check it seems that we're not having a code splitting here... We should have had a regression one day on this feature :(